assets/json/fish.json

```diff
@@ -0,0 +1,580 @@
+[
+  {
+    "name": "Charc Eel",
+    "thumb": "img/fish/charceel.png",
+    "unique": {
+      "name": "Charc Electroplax",
+      "thumb": "img/fish/part/electroplax.png",
+      "wiki": "http://warframe.wikia.com/wiki/Charc_Electroplax"
```

Comment:
While we're here, it'd be nice to switch to `https` wherever possible, but since this is out of scope, I'd be happy to make another PR later if you don't want to do this now.

Reply:
we're not loading insecure resources, it's a link to the wiki.

Reply:
it's a fast enough find and replace that i'll add it

Reply:
@dsinn changed